src/components/ChatRoomPage/Chat.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import styled from 'styled-components';
+import { IChat } from '../../interface/interface';
+import { useRecoilValue } from 'recoil';
+import { currentUserState } from '../../state/atom';
+import userData from '../../db/userData.json';
+
+const Chat = (chat: IChat) => {
+  const currentUser = useRecoilValue(currentUserState);
+  //chat.userId === currentUser.userId;
+
+  return (
+    <ChatContainer isActive={chat.userId === currentUser.userId}>
```

Comment:
```suggestion
    <ChatContainer isActive={isActive}>
```
위의 코멘트에 이어서 이런식으로 축약할 수 있을 것 같아요!